packages/kolibri-tools/jest.conf/setup.js

```diff
@@ -42,6 +45,11 @@ Vue.config.silent = true;
 Vue.config.devtools = false;
 Vue.config.productionTip = false;
 
+plugin_data['languageGlobals'] = {
+  languageCode: defaultLanguage.id,
+  availableLanguages: [defaultLanguage],
```

Comment:
This needs to be:
```suggestion
  languages: {defaultLanguage.id: defaultLanguage},
```

See line 211 in i18n.js, it expects a key of `languages` and for the value to be an object that is then assigned to the `availableLanguages`.

Reply:
Thanks for the explanation @rtibbles, the suggested change had syntax errors. I did a little debugging and came up with a change that passes the test cases. Please add your comments on that.